workflows/python/workflowscript.py

```diff
@@ -0,0 +1,139 @@
+# First import the packages that will be needed  
+from snowflake.connector.pandas_tools import write_pandas
+import snowflake.connector
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+import boto3
+import pickle
+import os
+import tempfile
+
+# Set some variables up to use them later
+bucket = os.environ['DATA_LAKE'] # For the data lake connection
+filepath = os.environ['DATA_LAKE_ROOT_PATH'] + 'datascience/' # For the datalake connection
+filename = 'predictions.csv' # Filename for the prediction data 
+key_model_save = os.environ['DATA_LAKE_ROOT_PATH'] + "datascience/model.pkl" # Model name
+
+
+## This function connects you to Snowflake
+def connect_to_db():
+    conn = snowflake.connector.connect(
+            user=os.environ['SNOWFLAKE_USERNAME'],
+            password=os.environ['SNOWFLAKE_PASSWORD'],
+            account=os.environ['SNOWFLAKE_ACCOUNT'],
+            warehouse=os.environ['SNOWFLAKE_WAREHOUSE'],
+            database=os.environ['SNOWFLAKE_DATABASE'], 
+            schema=os.environ['SNOWFLAKE_SCHEMA'])
+    return conn
+
+
+def read_from_db(connection_str, query):
+    cur = conn.cursor()
+    cur.execute(query)
+    df = cur.fetch_pandas_all()
+    return df
+
+# Printing statements so we have some logs
+print("Querying Snowflake data")
+
+# Query to get all of our raw data from STAGE
+query = """
+select * from STAGE.HOUSEPRICES
+"""
+conn = connect_to_db()
+df = read_from_db(conn, query)
```

Comment:
Nit. Variable name `df` could have been better.

Reply:
Leave this please